packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function transformer(file, api) {
```

Comment:
Please give the function a camel-case version of the file name.  A meaningful name will be easier to eyeball in stack traces.